compiler/testData/ir/irText/classes/initVal.kt

```diff
@@ -12,3 +12,11 @@ class TestInitValInInitBlock {
     }
 }
 
+class TestInitValInLambdaCalledOnce {
+    val x: Int
+    init {
+        1.run {
```

Comment:
`init { 1.run { } }` causes fir to fail with an error at `ConeClassifierLookupTag.toSymbol` because the tag is of unexpected type `ConeTypeVariableTypeConstructor`. Is there a way to mute this test for fir?

Reply:
You can implement a corresponding directive in `AbstractFir2IrTextTest` (something like `FIR_IGNORE`). 
@mglukhikh is this a known issue in FIR? Could you please investigate it, or forward it to someone else?

Reply:
Let me look at it...

Reply:
We investigated problem with @demiurg906. It's more or less clear what's happened, but it's non-trivial to fix it. I'd prefer to accept PR and mute test on teamcity, I'll fix it in a few days. If you do not like this way, we can return `null` from this `toSymbol` function temporarily, it will hide the problem.

Also it would be great to create separate IR test for this case (if possible) and not to add this case to existing test.

Reply:
cc @dnpetrov ^^^